Fix:
warning: 'ntohll' macro redefined [-Wmacro-redefined]
warning: 'htonll' macro redefined [-Wmacro-redefined]
Part of:
https://github.com/websupport-sk/pecl-memcache/commit/c051ca73ee98a4e5207799ea452470153695512f
--- memcache_binary_protocol.c.orig	2013-04-07 21:12:54.000000000 -0500
+++ memcache_binary_protocol.c	2021-12-23 06:11:28.000000000 -0600
@@ -34,6 +34,14 @@
 #include "memcache_pool.h"
 #include "ext/standard/php_smart_str.h"
 
+#ifdef htonll
+#undef htonll
+#endif
+
+#ifdef ntohll
+#undef ntohll
+#endif
+
 #if __BYTE_ORDER == __BIG_ENDIAN
 # define ntohll(x) (x)
 # define htonll(x) (x)
